include/ua_plugin_nodestore.h

```diff
@@ -188,7 +188,7 @@ typedef struct {
     UA_NODE_VARIABLEATTRIBUTES
     UA_Byte accessLevel;
     UA_Double minimumSamplingInterval;
-    UA_Boolean historizing; /* currently unsupported */
+    UA_Boolean historizing;
```

Comment:
The `read` and `write` services need to be extended so that users can actually interact with the historizing attribute.

Reply:
Done